doc/tutorials/using_sbl_on_up2.rst

```diff
@@ -99,26 +99,14 @@ An example of the configuration file ``uos.json``:
    the "Version" argument, ``"Version": 2****`` instead
    of ``"Version": 27230`` for example.
 
-Clone the source code of ``acrn-hypervisor`` for building and use 
-``v0.5`` release for example:
-
-.. code-block:: none
-   
-   $ cd ~
-   $ git clone https://github.com/projectacrn/acrn-hypervisor
-   $ cd acrn-hypervisor
-   $ git checkout v0.5
-
-.. note::
-   You can switch to other version with the command ``git checkout`` if needed.
 
 Build SOS and LaaG image:
    
 .. code-block:: none   
 
    $ sudo -s 
    # ./acrn-hypervisor/devicemodel/samples/up2/create-up2-images.sh --images-type all \
-      --clearlinux-version 27230 --laag-json uos.json --acrn-code-path /path/to/acrn-hypervisor/
+      --clearlinux-version 27230 --laag-json uos.json
```

Comment:
You removed the step that clones the acrn-hypervisor, so how do you get the ``create-up2-images.sh`` script for this command??